server/@types/express/index.d.ts

```diff
@@ -20,6 +20,9 @@ export declare global {
     interface JourneyData {
       educationSupportPlanDto?: EducationSupportPlanDto
       refuseEducationSupportPlanDto?: RefuseEducationSupportPlanDto
+      challengeDto?: ChallengeDto
+      conditionDto?: ConditionDto
+      strengthDto?: StrengthDto
```

Comment:
Added StrengthDto to the journeyData, along with the other DTOs we'll be working with in due course 👍